docs/static/breaking-changes.asciidoc

```diff
@@ -2,6 +2,23 @@
 == Breaking changes
 
 **Breaking changes in 5.0**
+
+Application Settings: Introduced a new way to configure application settings for Logstash through a settings.yml file. This file 
+is typically located in `LS_HOME/config`, or `/etc/logstash` when installed via packages. Logstash will not be able 
+to start without this file, so please make sure to pass in path.settings if you are starting Logstash manually 
+after installing it via a package (RPM, DEB).
+
+Release Packages: When Logstash is installed via DEB, RPM packages, it uses `/usr/share` and `/var` to install binaries and config files 
```

Comment:
Should these paths be more complete?  e.g. `/usr/share/logstash` and `/var/lib/logstash`?